src/CommunityToolkit.Maui.Core/Views/PaddedButton.macios.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using UIKit;
+
+namespace CommunityToolkit.Maui.Core.Views;
+
+/// <summary>
+/// <see cref="UIButton"/> with Left, Top, Right and Bottom Padding
+/// </summary>
+public class PaddedButton : UIButton
+{
+	/// <summary>
+	/// Initialize <see cref="PaddedButton"/>
+	/// </summary>
+	/// <param name="leftPadding"></param>
+	/// <param name="topPadding"></param>
+	/// <param name="rightPadding"></param>
+	/// <param name="bottomPadding"></param>
+	public PaddedButton(double leftPadding, double topPadding, double rightPadding, double bottomPadding)
```

Comment:
shouldn't change those values to be `nfloat` as well?

Reply:
it's easier to use double, because nfloat is specific to iOS

Reply:
@VladislavAntonyuk but we changed the `PaddedLabel` to accept `nfloat`
https://github.com/CommunityToolkit/Maui/pull/214/commits/4f5ad7dc9c0b441c2d34308b7fa81a7fbb993b1d

Reply:
Padding and CornerRadius use double. it would be easier to have double as parameter and cast it in one place

Reply:
> shouldn't change those values to be nfloat as well?
The difference is how the values are used in `PaddedLabel` vs `PaddedButton`.

`PaddedLabel` uses `public override CGSize IntrinsicContentSize` and `public override void DrawText` which both require `nfloat`.

`PaddedButton` uses `ContentEdgeInsets` which requires a `double.

Reply:
I see now, thanks @brminnick for the clarification.

Reply:
Correct. The cast from `double` to `nfloat` has to be done somewhere.

It's more honest for the constructor of `PaddedLabel` to require `nfloat` rather that letting it accept a `double` and immediately casting it to `nfloat` inside the constructor. 

Having `PaddedLabel`' constructor requiring `nfloat` let's the consumer of `PaddedLabel` decide how to cast their input type.

Reply:
Oh wait - I lied.

@pictos is right - `PaddedButton` uses `new UIContentEdgeInsets` which _does_ require `float`. I'll update `PaddedButton`'s constructor to use `nfloat` as well 👍